### ISSUE TITLE
Suggesting keras callback to handle tf.Dataset

### ISSUE THREAD

**Issue by Yannik1337** [wandb/client#1465]
I currently use a TF dataset. 
If I want to log gradients, I have to supply training data to the WandbCallback() in (x, y) tuples. I thus save (some) batches of my training set prior to the run as a numpy array to file, and load them "into" the callback.

As an easier solution, I suggest the callback to accept a tf.Dataset as input for training data.

**Comment by tyomhak** [comment]
Hey @Yannik1337 , thank you for the suggestion! 
cc @vanpelt

**Comment by cvphelps** [comment]
Hi, thanks for this feature request, good idea.

**Comment by ariG23498** [comment]
Hey @Yannik1337 
We have updated the package and now we support `tf.data.Datset` with the `WandbCallback`.
GIST: https://gist.github.com/ariG23498/02a9cace9399324a7c299203e1614f2c

**Comment by Yannik1337** [comment]
Thanks for this integration!
